### PR TITLE
Cherry pick #9110 and #10344 to Show Focus Indicator in Checkbox

### PR DESCRIPTION
**Pull request by Lijing29**
#### Pull request checklist

- [x] Addresses an existing issue: Cherry pick fix [#9110](https://github.com/OfficeDev/office-ui-fabric-react/pull/9110) and [#10344 ](https://github.com/OfficeDev/office-ui-fabric-react/pull/10344)
- [ ] Include a change request file using `$ yarn change`

#### Description of changes
Cherry pick [#9110](https://github.com/OfficeDev/office-ui-fabric-react/pull/9110) (released in [v6.182.1](https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.182.1)) and [#10344](https://github.com/OfficeDev/office-ui-fabric-react/pull/10344) (released in [v7.30.0](https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v7.30.0)) to show focus indicator in checkbox from Fabric 7 master branch to release version v6.203.3, which is based on the release v6.180.0

#### Focus areas to test
Checkbox accessbility

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10403)

### CODE REVIEW

**Comment by Lijing29** [comment]
@cliffkoh I have no idea why the change file is wrong. Do you have any suggestion about this? The change file in this PR is generated by running the script "**npm run change**". I have tried to run the script "**node ./scripts/convert-change-files.js**", it would delete the change file and generate a new one which looks like this:
image

**Comment by cliffkoh** [comment]
I *think* after it has been converted it probably need to be moved to the changes directory.. but @kenotron would know for sure as he wrote all of this (I would have to look at the code/debug). Sorry though that you're encountering some problems here.

**Comment by cliffkoh** [comment]
p.s. the problem specifically seesm to be with : 
image

which is still the old file format

**Comment by Lijing29** [comment]
The fix of [#9110](https://github.com/OfficeDev/office-ui-fabric-react/pull/9110) is released in v6.182.1. It seems this PR could not show the .style.ts.change out, as it is compared with the origin/6.0. 

Here is how I did the cherry-pick. If possible, please help to check it whether it is right or not.
The commit of the #9110 is https://github.com/OfficeDev/office-ui-fabric-react/commit/d0d926c5c415c4df00e3e537297e36cdb5efccdd
In my branch, I run:
image

Checking the file packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts, I found it has been changed:
image
 which is the same as the fix of the #9110:
image

In short, I think I have cherry-picked the .styles.ts change. However, the change isn't shown out. 
Please correct me if there is something wrong or missing. Thanks!
